pkg/providers/amifamily/custom.go

```diff
@@ -39,8 +42,8 @@ func (c Custom) UserData(_ *corev1beta1.KubeletConfiguration, _ []v1.Taint, _ ma
 	}
 }
 
-func (c Custom) DefaultAMIs(_ string) []DefaultAMIOutput {
-	return nil
+func (c Custom) AMIQuery(_ context.Context, _ ssm.Provider, _ string, _ string) (AMIQuery, error) {
```

Comment:
I think it's more clear that this should be nil when this is renamed to SSMPaths, or something of the like.

Reply:
Mentioned this in another comment, but `SSMPaths` doesn't really describe what this function returns. It does deal with SSM paths, but only internally. By the time we're returning details, any information about the SSM paths have been discarded. We're returning an `AMIQuery` which has the `image-id` filter and `KnownRequirements` populated.